openid-4-verifiable-presentations-1_0.md

```diff
@@ -252,10 +254,14 @@ This specification enables the Verifier to send both Presentation Definition JSO
 This specification defines the following new request parameters:
 
 `presentation_definition`:
-: A string containing a Presentation Definition JSON object. See (#request_presentation_definition) for more details. This parameter MUST be present when `presentation_definition_uri` parameter, or a `scope` value representing a Presentation Definition is not present.
+: A string containing a Presentation Definition JSON object. See (#request_presentation_definition) for more details. Exactly one of `vp_query` or `presentation_definition` or `presentation_definition_uri` MUST be present in the Authorization Request.
```

Comment:
What would be the normative requirements of an implementation with regard to what an RP must support vs a wallet, the way I would read this is they would need to support all options to be compliant?

Reply:
```suggestion
: A string containing a Presentation Definition JSON object. See (#request_presentation_definition) for more details. Exactly one of  the following parameters MUST be present in the Authorization Request:`vp_query`, `presentation_definition`, `presentation_definition_uri`, or a `scope` value representing a Presentation Definition.
```
the current description is not complete because it does not cover scopes